CONTRIBUTING.md

```diff
@@ -105,26 +162,80 @@ and how it works [here](https://github.com/probot/dco#how-it-works).
 To signify that you agree to the DCO for contributions, you simply add a line to each of your
 git commit messages:
 
-```
-Signed-off-by: Jane Smith <jane.smith@example.com>
-```
+  ```
+  Signed-off-by: Jane Smith <jane.smith@example.com>
+  ```
 
 In most cases, you can add this signoff to your commit automatically with the
 `-s` or `--signoff` flag to `git commit`. You must use your real name and a reachable email
 address (sorry, no pseudonyms or anonymous contributions). An example of signing off on a commit:
-```
-$ commit -s -m “my commit message w/signoff”
-```
+
+  ```
+  $ git commit -s -m “my commit message w/signoff”
+  ```
 
 To ensure all your commits are signed, you may choose to add this alias to your global ```.gitconfig```:
```

Comment:
@alphaX86 @shubham-tyagi @warunicorn19 Can anyone of you please share how you set up your .gitconfig file for the commits cuz I'm having some issues with DCO so thought I'll add these too in the doc so that it is much easier for others too on how to set up commits with the DCO sign up.
I checked some of your commits in other PR's and when you compare it with my commits they look different.

Reply:
In .gitconfig huh... let me check...

Reply:
![image](https://user-images.githubusercontent.com/59508546/140695436-b48c2eca-27cf-4c5f-bafb-2d70def3f16f.png)

This is mine actually and I didn't add any of the things xD Just enter -s that's all

Reply:
so you mean :
`$ git commit -s -m “New Change”`
That's it?
Cuz I used to do this : 
`$ git commit --signoff -m " Commited suggestions_2  Signed-off-by: Deepak Gupta <deepakgdkg1g8868@gmail.com>"`
and weirdly it doesn't look like how you all commit. It looks like this :
![Screenshot (1082)](https://user-images.githubusercontent.com/81250703/140696497-e2824038-a6c5-424b-9f72-371e53c69b7a.png)

Reply:
> Cuz I used to do this :
$ git commit --signoff -m " Commited suggestions_2 Signed-off-by: Deepak Gupta <deepakgdkg1g8868@gmail.com>"

Well you don't need to include that Sign-off line in commit message actually :') Git automatically makes one if you give -s or --signoff flag

Reply:
So @alphaX86, finally simply do this right?
`$ git commit -s -m “New Change”`

this confused me:
`$ commit -s -m “my commit message w/signoff”`

Reply:
Yes, I too just add -s in my git commit command. We don't need to include signoff in the actual commit message.
`git commit -s -m "<commit msg>"` is good enough.

Reply:
Aight thanks @alphaX86 @shubham-tyagi .